### PR TITLE
heal prod - update services to 2021.09

### PR DESCRIPTION
**Pull request by haraprasadj**
Link to Jira ticket if there is one:

### CODE REVIEW

**Comment by PlanXCyborg** [comment]
# healdata.org/manifest.json
## :warning: Services on branch
- hatchery

**Comment by PlanXCyborg** [comment]
# healdata.org/manifest.json
## :warning: Services on branch
- hatchery

**Comment by PlanXCyborg** [comment]
# healdata.org/manifest.json
## :warning: Services on branch
- hatchery
- metadata

**Comment by PlanXCyborg** [comment]
# healdata.org/manifest.json
## :warning: Services on branch
- hatchery

**Comment by PlanXCyborg** [comment]
# healdata.org/manifest.json
## :warning: Services on branch
- hatchery

**Review by nmetokishlubsky — DISMISSED** [review]

File: healdata.org/manifest.json [hunk 1]
```diff
@@ -8,25 +8,25 @@
...
     "fluentd": "fluent/fluentd-kubernetes-daemonset:v1.2-debian-cloudwatch",
     "hatchery": "quay.io/cdis/hatchery:feat_decouple_workspaces",
-    "indexd": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/indexd:2021.08",
-    "manifestservice": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/manifestservice:2021.08",
-    "metadata": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/metadata-service:1.6.1",
-    "peregrine": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/peregrine:2021.08",
-    "pidgin": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/pidgin:2021.08",
+    "indexd": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/indexd:2021.09",
+    "manifestservice": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/manifestservice:2021.09",
+    "metadata": "quay.io/cdis/metadata-service:1.6.1",
```
Comment (mfshao) [inline healdata.org/manifest.json:20]: let's pull this from ecr as well

Comment (haraprasadj) [reply healdata.org/manifest.json]: done

**Comment by PlanXCyborg** [comment]
# healdata.org/manifest.json
## :warning: Services on branch
- hatchery

**Comment by PlanXCyborg** [comment]
# healdata.org/manifest.json
## :warning: Services on branch
- hatchery

**Review by nmetokishlubsky — APPROVED** [review]

**Pull request merged**